portal/views/assessment_engine.py

```diff
@@ -1606,6 +1606,9 @@ def present_needed():
         return redirect('/')
 
     url = url_for('.present_assessment', **args)
+    auditable_event("present assessment url {}".format(url),
```

Comment:
This seems like debugging info, is it really necessary to save to the audit log?